.nais/naiserator-prod.yaml

```diff
@@ -81,6 +81,8 @@ spec:
   env:
     - name: KTOR_ENV
       value: "production"
```

Comment:
Trengs det også en tilsvarende accessPolicy som er lagt til for dev her?

Reply:
Ja, det kan godt være. I første omgang må vi få det til å virke i preprod - så tenkte å vente med prod-tilpasningene.

Reply:
Ok, skjønner 👍